integration-tests/check-subscriptions

```diff
@@ -254,6 +264,45 @@ class TestSubscriptions(SubscriptionsCase):
         # find another one that is subscribed
         self.wait_subscription(PRODUCT_SHARED, True)
 
+        # unregister
+        b.click(unregister_button_sel)
```

Comment:
Is there any point in unregistering here if we don't check whether or not it has worked?

Reply:
Good point, there is no test if unregister works as expected. I will add the test here. OK?

Reply:
Done: please look at line 271.